test/v2/fixtures/retrieveBalanceAndTransactions/FinancialDetailsItemFixture.scala

```diff
@@ -22,7 +22,7 @@ import v2.models.response.retrieveBalanceAndTransactions.FinancialDetailsItem
 trait FinancialDetailsItemFixture {
```

Comment:
Can we also update the FinancialDetailsItem spec? I believe the rules on how to map the values in "statisticalDocument" have been updated in the spec now. With the current spec, an error should be thrown if statisticalDocument is any string that is not "Y" or "N"

Reply:
Yes, I have updated it now.